swig/sundials/fsundials_core_mod.i

```diff
@@ -0,0 +1,29 @@
+// ---------------------------------------------------------------
+// SUNDIALS Copyright Start
+// Copyright (c) 2002-2024, Lawrence Livermore National Security
+// and Southern Methodist University.
+// All rights reserved.
+//
+// See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// SUNDIALS Copyright End
+// ---------------------------------------------------------------
+// Swig interface file
+// ---------------------------------------------------------------
+
+%module fsundials_core_mod
+
+%include "fsundials_types.i"
+%include "fsundials.i"
+//%include "fsundials_errors.i"
```

Comment:
Remove or does something needed to be added for error handling?

Reply:
I think it can be removed